src/main/java/se/bjurr/violations/lib/example/MyClass.java

```diff
@@ -6,6 +6,7 @@
  void npe(String a, String b) {
   if (a == null) {
    System.out.println();
+   System.out.println();
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.blocks.EmptyBlockCheck
**Severity**: INFO
**File**: /home/bjerre/workspace/violations/violation-comments-to-github-plugin/work/workspace/pipe/src/main/java/se/bjurr/violations/lib/example/MyClass.java L9

Must have at least one statement.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1658594070>*